impedance/models/circuits/elements.py

```diff
@@ -377,7 +378,7 @@ def T(p, f):
         else:
             sinh.append(1e10)
```

Comment:
I can't comment on the whole block but L374-379 make me a bit uneasy. L376 checks whether values of `beta` are larger than 100, but I believe it only checks the real part, even though `beta` will always be complex. Granted, the function only blows up when the real part is large, but I think we should make the check explicit. 

Next, if `beta.real` is >100, we sub in `1e10` to avoid overflow errors. However, I think that fallback value is too low, and it's missing the imaginary component. For example, if we look at the overflow test on L89-92 of `test_circuit_elements.py`, we're using `params=[1, 2, 50, 100]` (I have no idea if those are reasonable or even physically possible). 

If we use a frequency of 35 Hz, then `beta = 74.15 +74.14j` and `np.sinh(beta) = (2.48e+31-7.58e+31j)`. If we instead use a frequency of  80 Hz then `beta = 112.10+112.09j` and `np.sinh(beta) = 1e10`. It seems weird that the output should be so much lower. I don't have a specific replacement number in mind, but I worry that 1e10 gets into the range of impedances people may actually be trying to fit?

Idk... pragmatically speaking 1e10 is 10GOhm so maybe it's not something to really worry about?

I also have problems with overflow in tanh that don't happen in the CI testing pipeline. I'm guessing that's because I'm on Windows, but the pipeline uses Unix? I've fixed this locally by replacing L374-379 with
```python
    sinh, tanh = [], []
    for x in beta:
        # beta will be dtype np.complex128 by default, so real and imag components are each float64

        if x.real < 100:
            sinh.append(np.sinh(x))
            tanh.append(np.tanh(x))
        else:
            sinh.append(1e10)
            tanh.append(1 + 0j)
```